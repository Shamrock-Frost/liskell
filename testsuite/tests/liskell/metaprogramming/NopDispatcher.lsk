(defmodule NopDispatcher _ (LskToHs SrcLoc LskParseTree LskTransformationMonad Quote))

(define (nop-dispatcher k pt)
    (if (== pt (PSym noSrcSpan "x" ""))
	(k (PSym noSrcSpan "y" ""))
	(k pt)))

(defenv (lambda ((LskEnv e p t d)) (return (LskEnv (nop-dispatcher e) p t d))))

(define y 12)
(define foo x)
(define main
    (print foo))
