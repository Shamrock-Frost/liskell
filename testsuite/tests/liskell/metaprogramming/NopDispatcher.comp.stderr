
NopDispatcher.lsk:1:79:
    Failed to load interface for `Quote':
      Use -v to see a list of the files searched for.
